research/src/pages/select.mdx

```diff
@@ -0,0 +1,132 @@
+---
+menu: Components
+name: Select
+path: /components/select
+research: /components/select.research
+---
+
+import Image from '../components/image'
+import SelectAnatomy from '../components/select-anatomy'
+
+The `<select>` element is a control that allows the user to select from a set of potential options.
+
+## Anatomy
+
+### Diagram
+
+<SelectAnatomy />
+
+### Part Definitions
+
+`::host` — Root element that contains the button and the pop-up.
+
+`::part-button` — An interactive element that toggles the appearance of the pop-up. This element has a tab stop.
+
+`::part-pop-up` — An interactive element containing the set of options the user can select from.
+
+### Events
+
+- `::host` Listening for `toggle-pop-up` event to hide/show `::part-pop-up`
+
+- `::part-button` Listens to `click` event and dispatches `toggle-pop-up` event
+
+- `::part-button` Listening for `option-change` event to updateselected value shown
+
+- `::part-pop-up` Listening for `option-change` event to dispatch `toggle-pop-up` event
+
+- `::part-button` Dispatches custom event `split-button-clicked`
+
+## Accessibility
+
+### Keyboard
+
+Keyboard users can operate the `<select>` as follows:
+
+- Focus on the `<select>` element using the `Tab` key (there is a tab stop on the button). The control can be
+  focused by default by setting the `autofocus` attribute.
+- Having focus, but with the pop-up still closed, users can:
+  - Change the selection to the next option using the `Down` arrow
+  - Change the selection to the previous option using the `Up` arrow
+  - Change the selection to the first option starting with a certain sequence by typing that sequence
+  - Cycle the selection through options starting with a certain letter by continually typing that letter
+- Having focus, the user can open the popup using the `Enter` key or the `Space` key. The popup can be dismissed using
+  the `Enter` key or the `Esc` key.
+- Having focus and with the pop-up open, the user can select options in the same manner as above. As the user traverses
+  the options, the currently-selected option is always visible and highlighted, with the menu scrolling as necessary.
+
+### Assistive Technology
+
+Implements the [combobox role](https://www.w3.org/TR/core-aam-1.1/#role-map-combobox) per the
+[HTML AAM spec](https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings).
+
+### Forced Colors
+
+Placeholder for now, dependent on standardization on parts definitions.
+
+## States / Properties
+
+### &ltselect&gt
```

Comment:
for some reason markdown makes headers render the &lt; and doesn't convert them - so for headers use <select>